tests/Resources/ti.geolocation.test.js

```diff
@@ -0,0 +1,289 @@
+		should(Ti.Geolocation).have.a.property('setTrackSignificantLocationChange').which.is.a.Function;
+		Ti.Geolocation.setTrackSignificantLocationChange(true); // defaults to false
+		should(Ti.Geolocation.trackSignificantLocationChange).eql(true);
+	});
+
+	// Methods
+
+	it('#forwardGeocoder()', function (finish) {
+		this.timeout(6e4); // 60 sec
+
+		should(Ti.Geolocation.forwardGeocoder).be.a.Function;
+		Ti.Geolocation.forwardGeocoder('440 N Bernardo Ave, Mountain View', function (data) {
+			try {
+				should(data).have.property('success').which.is.a.Boolean;
+				should(data.success).be.eql(true);
+				should(data).have.property('code').which.is.a.Number;
+				should(data.code).be.eql(0);
+				should(data.latitude).be.approximately(37.387, 0.002); // iOS gives: 37.38605, Windows does 37.3883645
+				should(data.longitude).be.approximately(-122.065, 0.02); // WIndows gives: -122.0512682, iOS gives -122.08385
+				finish();
+			} catch (err) {
+				finish(err);
+			}
+		});
+	});
+
+	// FIXME The address object is different from platform to platform! https://jira.appcelerator.org/browse/TIMOB-23496
+	it('#reverseGeocoder()', function (finish) {
+		this.timeout(6e4); // 60 sec
+
+		should(Ti.Geolocation.reverseGeocoder).be.a.Function;
+		Ti.Geolocation.reverseGeocoder(37.3883645, -122.0512682, function (data) {
+			try {
+				should(data).have.property('success').which.is.a.Boolean;
+				should(data.success).be.eql(true);
+				should(data).have.property('code').which.is.a.Number;
+				should(data.code).be.eql(0);
+				// FIXME error property is missing altogether on success for iOS...
+				// should(data).have.property('error'); // undefined on success, holds error message as String otherwise.
+				should(data).have.property('places').which.is.an.Array;
+				// FIXME Parity issues!
+				if (utilities.isAndroid()) {
+					should(data.places[0].postalCode).be.eql('94043');
+					should(data.places[0]).have.property('latitude').which.is.a.String;
+					should(data.places[0]).have.property('longitude').which.is.a.String;
+				} else {
+					should(data.places[0].zipcode).be.eql('94043');
+					should(data.places[0]).have.property('latitude').which.is.a.Number; // docs say String!
+					should(data.places[0]).have.property('longitude').which.is.a.Number; // docs say String!
+				}
+				should(data.places[0].country).be.eql('USA');
+				should(data.places[0].state).be.eql('California');
+				should(data.places[0].country_code).be.eql('US');
+				should(data.places[0]).have.property('city').which.is.a.String;
+				should(data.places[0]).have.property('address').which.is.a.String;
+
+				finish();
+			} catch (err) {
+				finish(err);
+			}
+		});
+	});
+
+	it('#getCurrentPosition()', function () {
+		should(Ti.Geolocation).have.a.property('getCurrentPosition').which.is.a.Function;
+	});
+
+	it('#getCurrentHeading()', function () {
+		should(Ti.Geolocation).have.a.property('getCurrentHeading').which.is.a.Function;
+	});
+});
```

Comment:
<!--
  1 failure:  tests/Resources/t...
  0 warning: 
  
  
  DangerID: danger-id-default;
  File: tests/Resources/ti.geolocation.test.js;
  Line: 289;
-->
- :no_entry_sign: tests/Resources/ti.geolocation.test.js line 289 – Newline required at end of file but not found. (eol-last)